templates/solutions/ai-infrastructure/index.html

```diff
@@ -0,0 +1,454 @@
+        </p>
+        <p class="p-section--shallow">
+          Our modular MLOps platform equips you with everything you need to bring your models all the way from experimentation to production. With easy access to key tools, you can move quickly and at scale.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://ubuntu.com/ai/mlops">Discover our MLOps platform&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+  </div>
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="https://ubuntu.com/engage/mlops-toolkit">Jumpstart your AI journey, download the MLOps toolkit&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Security across all layers of the stack</h2>
+      </div>
+      <div class="col">
+        <p class="p-section--shallow">
+          Canonical maintains and supports all the open source tooling in your AI infrastructure stack, including Kubernetes and open source cloud management solutions like OpenStack. Fast-track compliance and run AI projects securely with critical CVEs fixed in under 24h on average.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="">Explore secure open source&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Confidential AI</h2>
+      </div>
+      <div class="col">
+        <p>
+          Confidential AI on Ubuntu protects data in use at the hardware level. Building on Ubuntu confidential VMs, you can now safeguard your sensitive data and intellectual property with a hardware-rooted execution environment that spans both the CPU and GPU.
+        </p>
+        <p>Ubuntu confidential VMs are available on Azure, Google Cloud and AWS.</p>
+        <p class="p-section--shallow">
+          You can enable confidential computing in the datacenter and at the edge thanks to Ubuntu Intel TDX build, which comes with all the required pieces for both the guest and host.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://ubuntu.com/confidential-computing/ai">Read more about confidential AI&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+  </div>
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="https://canonical.com/solutions/infrastructure">Build your cloud with Canonical&nbsp;&rsaquo;</a>
```

Comment:
No need for the full path for local links
```suggestion
        <a href="/solutions/infrastructure">Build your cloud with Canonical&nbsp;&rsaquo;</a>
```